code/src/main/webapp/JavaScript/approval.js

```diff
@@ -0,0 +1,20 @@
+function removeLines() {
+  document.getElementById("wrapper-background").classList.add("remove")
+}
+
+function sendListToServlet() {
+  const params = new URLSearchParams();
+  params.append('idToken', localStorage.idToken);
+
+  let emails = [];
+  for (let contact of document.getElementsByClassName('contact')) {
+    const input = contact.getElementsByClassName("container")[0].getElementsByTagName("input")[0];
+    if (input.checked) {
+       emails.push(contact.getElementsByClassName("email")[0].innerText);
+    }
```

Comment:
Overall optional question: do you want to do the language selection at this point? or wait for a follow up PR?

Reply:
let's wait